client/app/components/dashboards/TextboxDialog.jsx

```diff
@@ -40,11 +40,30 @@ function TextboxDialog({ dialog, isNew, ...props }) {
     });
   }, [dialog, isNew, text]);
 
+  const confirmDialogDismiss = useCallback(() => {
+    const originalText = props.text;
+    if (text !== originalText) {
+      Modal.confirm({
+        title: "Quit editing?",
+        content: "Changes you made so far will not be saved. Are you sure?",
+        okText: "Yes, discard changes",
```

Comment:
```suggestion
        okText: "Yes, quit",
```

Lastly, just to make it aligned with the heading.